cmd/policy/main.go

```diff
@@ -0,0 +1,397 @@
+	if len(argsRest) != 0 {
+		return fmt.Errorf("unexpected arguments: %v (see -h for usage)", argsRest)
+	}
+
+	policies, err := pm.ListPolicies(tenantID)
+	if err != nil {
+		return err
+	}
+
+	for _, pentry := range policies {
+		fmt.Println(pentry.TenantID, pentry.TokenFormatName)
+	}
+
+	return nil
+}
+
+func runGetCommand(config *common.Config, args []string, pm *policy.Manager, logger *zap.Logger) error {
+	var tenantID int
+	var outPath string
+
+	addFlags := flag.NewFlagSet("add", flag.ExitOnError)
+	addFlags.IntVar(&tenantID, "t", 1, "TenantID to which the policies should be added.")
+	addFlags.StringVar(&outPath, "o", "policy.zip", "Output zip path.")
+
+	if err := addFlags.Parse(args); err != nil {
+		return err
+	}
+
+	argsRest := addFlags.Args()
+	if len(argsRest) == 0 {
+		return fmt.Errorf("token format(s) not specified (see -h for usage)")
+	}
+
+	var formats []common.TokenFormat
+	for _, fmtName := range argsRest {
+		format, err := common.TokenFormatFromString(fmtName)
+		if err != nil {
+			return err
+		}
+
+		formats = append(formats, format)
+	}
+
+	var policies []*common.Policy
+	for _, format := range formats {
+
+		policy, err := pm.GetPolicy(tenantID, format)
+		if err != nil {
+			return fmt.Errorf(
+				"could not get policy for tenant %d, format %q: %v",
+				tenantID,
+				format.String(),
+				err,
+			)
+		}
+
+		policies = append(policies, policy)
+	}
+
+	return common.WritePoliciesToPath(policies, outPath)
+}
+
+func runAddCommand(config *common.Config, args []string, pm *policy.Manager, logger *zap.Logger) error {
+	var tenantID int
+	var force bool
+
+	addFlags := flag.NewFlagSet("add", flag.ExitOnError)
+	addFlags.IntVar(&tenantID, "t", 1, "TenantID to which the policies should be added.")
+	addFlags.BoolVar(&force, "f", false, "If specified, an existing policy will be overwritten, if it exists.")
+
+	if err := addFlags.Parse(args); err != nil {
+		return err
+	}
+
+	argsRest := addFlags.Args()
+	if len(argsRest) == 0 {
+		return fmt.Errorf("policy path not specified (see -h for usage)")
+	} else if len(argsRest) > 1 {
+		return fmt.Errorf("unexpected arguments: %v (see -h for usage)", argsRest[1:])
+	}
+
+	policiesZipPath := argsRest[0]
+
+	policies, err := common.ReadPoliciesFromPath(policiesZipPath)
+	if err != nil {
+		return fmt.Errorf("problem reading policies: %v", err)
+	}
+
+	if len(policies) == 0 {
+		return fmt.Errorf("no policies found inside %v", policiesZipPath)
+	}
+
+	for _, policy := range policies {
+
+		_, err := pm.GetPolicy(tenantID, policy.TokenFormat)
+		if err == nil { // policy exists
+			if force {
+				err = pm.DeletePolicy(tenantID, policy.TokenFormat)
+				if err != nil {
+					return fmt.Errorf(
+						"could not remove existing policy for tenantID %d and format %q: %v",
+						tenantID,
+						policy.TokenFormat.String(),
+						err,
+					)
+				}
+			} else {
+				return fmt.Errorf(
+					"policy for tenant 1 and fromat %q already exists; use -f to overwrite",
```

Comment:
typo: `"policy for tenant %d [...]", tenantID)`

Reply:
D'oh. Thanks!